### ISSUE TITLE
Issue with parsing the coordinates in array format

### ISSUE THREAD

**Issue by gharibi** [caseycesari/GeoJSON.js#36]
The main assumption of the data is to be like:
```
var data = [
  { name: 'Location A', category: 'Store', street: 'Market', lat: 39.984, lng: -75.343 },
  { name: 'Location B', category: 'House', street: 'Broad', lat: 39.284, lng: -75.833 },
  { name: 'Location C', category: 'Office', street: 'South', lat: 39.123, lng: -74.534 }
];
```

when the coordinates are within an array as below, the generated GeoJSON is not valid:

```
var data = [
  { name: 'Location A', category: 'Store', street: 'Market', coordinates: [39.984, -75.343] },
  { name: 'Location B', category: 'House', street: 'Broad', coordinates: [39.284, -75.833] },
  { name: 'Location C', category: 'Office', street: 'South', coordinates: [39.123, -74.534] }
];
```

I tried to parse the above-mentioned dataset as below and it didn't work either:
`GeoJSON.parse(data, {Point: ['coordinates']});`

**Comment by trandiepbang** [comment]
Same :(

**Comment by KaiBot3000** [comment]
Same; I ran into this trying to parse `LineStrings`, which have coordinate arrays.

**Comment by danvucore** [comment]
Right, because geojson format by GeoJSON standard 
[follow here](https://gis.stackexchange.com/questions/54065/leaflet-geojson-coordinate-problem)

**Comment by caseycesari** [comment]
@gharibi Assigned. Thanks!

**Comment by gharibi** [comment]
Done. Created a PR for it 😉

**Comment by caseycesari** [comment]
Fixed in #39
